apps/web/src/components/TokenImage/index.tsx

```diff
@@ -32,8 +27,8 @@ export const TokenPairImage: React.FC<React.PropsWithChildren<TokenPairImageProp
   const chainLogo = withChainLogo ? getChainLogoUrlFromChainId(primaryToken.chainId) : undefined
   return (
     <UIKitTokenPairImage
-      primarySrc={getImageUrlFromToken(primaryToken)}
-      secondarySrc={getImageUrlFromToken(secondaryToken)}
+      primarySrc={getCurrencyLogoSrcs(primaryToken)[0]}
```

Comment:
getCurrencyLogoSrcs always returns an array so it is safe to check with index directly